src/main/archetype/ui.apps/src/main/content/jcr_root/apps/__appsFolderName__/clientlibs/clientlib-base/css.txt

```diff
@@ -0,0 +1,15 @@
+${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}${hash}
```

Comment:
I hash sign a problem with mvn archetype so we need to encode it?

Reply:
The archetype uses hashes to indicate variables, so yes.

Reply:
If the file is anyway empty, let's remove that horrible header ... or even remove the file? (not sure if it's ok to not have that file in a clientlib)

Reply:
I double checked and the files can be empty and the clientlib is still properly generated. So I removed the ugly header.

Reply:
👍